Formula/entr.rb

```diff
@@ -35,3 +40,17 @@ def install
     assert_equal "New File", pipe_output("#{bin}/entr -n -p -d echo 'New File'", testpath).strip
   end
 end
+__END__
```

Comment:
```suggestion

__END__
```
Most other formulae have whitespace here for embedded patches. (I forget why, but let's try to be consistent...)